Scraping Script Workspace.ipynb

```diff
@@ -67,42 +72,83 @@
     "    driver.get(url)\n",
     "    html = driver.page_source\n",
     "    soup = bs(html)\n",
-    "    page_download(soup)"
+    "    page_empty = page_download(soup)\n",
+    "    return page_empty"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 122,
-   "id": "66171a24",
+   "execution_count": 140,
+   "id": "09199600",
    "metadata": {},
    "outputs": [],
    "source": [
     "def scrape_tour(base_url, tour_code, driver):\n",
     "    url = base_url + tour_code + '?page='\n",
-    "    scrape_page(url, 10, driver)"
+    "    page_empty = False\n",
+    "    i = 1\n",
+    "    while not page_empty:\n",
+    "        page_empty = scrape_page(url, i, driver)\n",
+    "        i += 1\n",
+    "        print(i)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "id": "875cd2c3",
+   "execution_count": 125,
+   "id": "5b8efdff",
    "metadata": {},
    "outputs": [],
    "source": [
-    "def scrape_slippi(base_url, driver):\n",
-    "    url = "
+    "# def scrape_slippi(base_url, driver):\n",
+    "#     url = "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "id": "5237331f",
+   "execution_count": 143,
+   "id": "259568ba",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2\n",
+      "3\n",
+      "4\n"
+     ]
+    }
+   ],
    "source": [
     "driver = webdriver.Firefox()\n",
     "scrape_tour(_URL,'0x7', driver)"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 131,
+   "id": "59f0d578",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<a href=\"/downloads\">downloads page</a>]"
+      ]
+     },
+     "execution_count": 131,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "driver = webdriver.Firefox()\n",
+    "driver.get('https://slippi.gg/tournament/0x7?page=14')\n",
+    "html = driver.page_source\n",
+    "soup = bs(html)\n",
+    "soup.findAll('a')"
+   ]
```

Comment:
I don't know anything about webscraping but this is super cool + good u included it